App/Logic/OnboardingLogic.swift

```diff
@@ -328,8 +328,31 @@ extension Logic.Onboarding {
       state.user.province = self.province
     }
   }
+    
+  struct SetUserCountries: AppStateUpdater {
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/OnboardingLogic.swift;
  Line: 333;
-->

- :warning: swiftlint (0.39.2): Lines should not have trailing whitespace. (trailing_whitespace)